src/java.base/share/classes/jdk/internal/javac/PreviewFeature.java

```diff
@@ -68,7 +68,7 @@ public enum Feature {
         SWITCH_PATTERN_MATCHING(),
         @JEP(number=432, title="Record Patterns", status="Second Preview")
         RECORD_PATTERNS,
-        @JEP(number=436, title="Virtual Threads", status="Second Preview")
+        // not used
         VIRTUAL_THREADS,
```

Comment:
The javadoc `Preview` page lists the preview features (and the JEPs) that are part of a release. For example, for JDK 20 the page is here https://docs.oracle.com/en/java/javase/20/docs/api/preview-list.html. 

In a JDK image that's built out of this PR, does the missing `@JEP` annotation on this enum value cause issues in javadoc image generation or presentation on that page?

Reply:
> In a JDK image that's built out of this PR, does the missing `@JEP` annotation on this enum value cause issues in javadoc image generation or presentation on that page?

This has been looked at a few times and we didn't find any issues. Ideally the constant would be removed but we are forced to leave it for now.